gateway/gateway/Cargo.toml

```diff
@@ -49,6 +49,8 @@ tokio-stream = "0.1.11"
 tonic = { version = "0.8", features = ["transport", "tls"] }
 tower-http = { version = "0.3.5", features = ["cors", "auth"] }
 url = { version = "2.3.1", features = ["serde"] }
+tonic_openssl_lnd = { git = "https://github.com/justinmoon/tonic_openssl_lnd", rev = "68b97570" }
```

Comment:
consider using [this fork of tonic_lnd](https://github.com/okjodom/tonic_lnd/tree/routerrpc-feature). It includes he important bits you need from `tonic_openssl_lnd`: maintained tonic ssl, i.e.
- tonic_lnd fix for ip address issue. fixed by moving to openssl
- lnd router-rpc

Additionally:
- Includes rust features, so you can pick just the LND rpc api features you need for LND support
- Is on path for proper dependency integration so we have prod ready LND integration

Reply:
Nice. Will try this. Feel free to try it yourself, too.

Reply:
https://github.com/justinmoon/fedimint/pull/48

Reply:
Merged